index.js

```diff
@@ -507,6 +515,60 @@ app.get('/api/user/:id', async (req, res) => {
   }
 });
 
+
+
+// register new user
+app.post('/api/register', async (req, res) => {
+  var { name, email, phone_number, password } = req.body;
+
+  if (email == "" || name == "" || password == "") {
+    res.status(400).json({error: 'All_FIELDS_ARE_REQUIRED'});
+    return;
+  }
+
+  if (phone_number != undefined && phone_number != "" && phone_number.length != 10 ) {
```

Comment:
```suggestion
  if (phone_number && phone_number.length != 10 ) {
```
similar to the above comment